ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -1,369 +1,199 @@
 const express = require('express')
-const crypto = require('crypto')
-const app = express();
-const logger = require('./logs/logger');
-const ehsm_napi = require('./ehsm_napi');
-const { ehsm_kms_params, ehsm_keyspec_t, ehsm_keyorigin_t} = require('./ehsm_kms_params.js')
-const userInfo = require('./user_info')
-app.use(express.json());
 
-const PORT = process.argv.slice(2)[0] || 9000;
+const logger = require('./logger')
```

Comment:
Why commit this file? it's in #47 .

Reply:
The NAPI_RA_HANDSHAKE_MSG0 function is added on the basis of 47